tcms/tests/factories.py

```diff
@@ -260,7 +260,7 @@ class Meta:
     summary = factory.LazyAttribute(lambda obj: 'Summary of bug %s' % obj.bug_id)
     description = ''
     bug_system = factory.LazyFunction(lambda: BugSystem.objects.all()[0:1][0])
-    case_run = factory.SubFactory('tests.TestCaseRunFactory')
+    case_run = factory.SubFactory('tcms.tests.factories.TestCaseRunFactory')
```

Comment:
Use `TestCaseRunFactory` as a variable name instead of string. Because it is defined later in the module you have to move the entire class definition after the definition of TestCaseRunFactory.